aries.md

```diff
@@ -0,0 +1,23 @@
+* Connection:
```

Comment:
Good, how about adding links to corresponding RFCs?

Reply:
Added